boot-vault.platform.yaml.template

```diff
@@ -46,3 +46,38 @@ controllerbuild:
     repository: gcr.io/jenkinsxio/builder-go
     tag: $VERSION
 {{- end }}
+
```

Comment:
just so that I understand better, where and how is this template file used? Any chance it could be moved to somewhere else (I realize that you are just adding things in this case, but I really would love to reduce the clutter in the root directory of the project)

Reply:
What's about these `CODECOV_TOKEN: $CODECOV_TOKEN` lines? Could they be removed? I think they might be leftovers from the codecov experiment.

Reply:
Yeah, I can definitely move these and nuke the codecov stuff.